octoprint_domoticz/templates/domoticz_settings.jinja2

```diff
@@ -91,3 +101,17 @@
 		<a href="#" class="btn" data-dismiss="modal" aria-hidden="true">{{ _('Close') }}</a>
 	</div>
 </div>
+<script type="text/javascript">
+    function copyTextToClipboard(text) {
+        if (navigator.clipboard) {
+            navigator.clipboard.writeText(text);
+        } else {
+            var textarea = document.createElement("textarea");
+            textarea.value = text;
+            document.body.appendChild(textarea);
+            textarea.select();
+            document.execCommand("copy");
+            document.body.removeChild(textarea);
+        }
+    }
+</script>
```

Comment:
No need to add copy command, a global function is available within OctoPrint core to do this for us already.
```
copyToClipboard(text)
```